flask-app/app.py

```diff
@@ -47,19 +54,103 @@ def proxy_request_to_brouter(parameters):
     return Response(response.content, response.status_code, headers)
 
 
+def get_regions(model):
+    db_session = DBSession()
```

Comment:
Please try to avoid creating new sessions. One session per endpoint should be enough. You can pass the session to helper functions like this one

Reply:
I got rid of such practice in cases endpoints' functions.
I suppose creating `DBSession` in `get_region` / `get_regions` is okay since each function using either of those creates only one session. In case this changes, we could indeed pass `db_session` as an argument.